plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java

```diff
@@ -119,19 +124,26 @@ public void done(IJobChangeEvent event) {
     }
   }
 
+  private String getConsoleName(String project) {
+    Date now = new Date();
```

Comment:
Yes, I also prefer JodaTime. However it is not a dependency of our plugin yet, so I'm not going to add it just to get the current time (which as you noted doesn't really matter if we do it via `java.util.Date`)